lib/addMethod/globalize/object.js

```diff
@@ -9,13 +9,14 @@ var substitute = require('../substitute');
 
 module.exports = function (key, config, params) {
 
-  var subbedGlobalObject = substitute(this._globalOptions[key], params || {});
-  var subbedObject       = substitute(config[key] || {}, params || {});
+    params = params || {};
 
-  if (config.globals === false || !_.isObject(subbedObject)) {
-  	return subbedObject;
-  } else {
-  	return _.defaultsDeep(subbedObject, subbedGlobalObject);
-  }
+    var subbedGlobalObject = substitute(this._globalOptions[key], params);
+    var subbedObject       = substitute(config[key] || {}, params);
+
+    if (config.globals === false)
```

Comment:
Can we keep the curly braces in here, not a fan of removing them when its not a website payload which needs to be minimised.